backend/src/test/java/com/redhat/cloud/notifications/routers/EventServiceTest.java

```diff
@@ -469,7 +470,16 @@ void testAllQueryParams() {
                 .chain(runOnWorkerThread(() -> {
                     // TODO Temp test, remove ASAP
                     System.setProperty("notifications.event-service.legacy-mode", "false");
-                    return getEventLogPage(defaultIdentityHeader, null, null, null, null, null, null, null, null, null, null, false);
+                    getEventLogPage(defaultIdentityHeader, null, null, null, null, null, null, null, null, null, null, false);
+                    System.setProperty("notifications.event-service.legacy-mode", "true");
+                    given()
+                            .basePath(API_INTERNAL)
+                            .when()
+                            .get("/denormalizedEvents/migrate")
+                            .then()
+                            .statusCode(200);
+                    System.setProperty("notifications.event-service.use-denormalized-events", "true");
```

Comment:
Is this really changing the mode to use the normalized events? Are the queries indeed different in this test?

Reply:
Yes, all queries are different with the boolean set to `true`.

Reply:
FYI I ran all the tests locally with `notifications.event-service.use-denormalized-events=true` and made sure they passed.